site/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "hacker-news-pwas",
+  "description": "Hacker News readers as Progressive Web Apps",
+  "scripts": {
+    "build": "bundle exec jekyll build",
+    "dev": "bundle exec jekyll serve",
```

Comment:
Do you think we need a command to run the site without a service worker? 🤔  I would think anytime a user wants to boot the site up - there's no reason to not include the service worker. I'm okay with removing this separation and having a single `dev` command thats generates the service worker and boots the site up.

Thinking out loud here however, I'm perfectly okay with leaving it as is if you think it's better :)